source/extensions/stat_sinks/common/statsd/statsd.cc

```diff
@@ -38,34 +40,73 @@ void UdpStatsdSink::WriterImpl::write(const std::string& message) {
   Network::Utility::writeToSocket(*io_handle_, &slice, 1, nullptr, *parent_.server_address_);
 }
 
+void UdpStatsdSink::WriterImpl::writeBuffer(Buffer::Instance* data) {
```

Comment:
nit: prefer reference argument instead of pointer argument.

UdpStatsdSink::WriterImpl::writeBuffer(Buffer::Instance& data) {

Reply:
Fixed.